python/requirements.txt

```diff
@@ -0,0 +1,2 @@
+vertx-eventbus-bridge-starter
```

Comment:
> Why not start by including it directly in this project, and then separating it when needed?

The intention of the test project is not only for Python client, but also for other clients. You are right that it makes no sense to have a test only dependency declared in the `requirements.txt`.

Reply:
> The intention of the test project is not only for Python client, but also for other clients.

I totally understand the intention, but the quality isn't up to speed, and it isn't because the first use case here isn't mature. This is normal. Including the test project here for the first step seems safer to me, and it doesn't preclude it from being separated later when it is more mature and ready to be shared by other projects.

Reply:
> it doesn't preclude it from being separated later when it is more mature and ready to be shared by other projects.

OK, I think it makes sense, thank you :)  The test project includes both Java code of the bridge server, and the Python code on how to start the bridge, I will move the Python one only to the test in this project, and leave the Java one as where it is to make this project clean. 

> `requirements.txt` is meant for development dependencies. Runtime dependencies are often specified using `setuptools` in a `setup.py` file

I see, thank you :) This `vertx-eventbus-client` should not depend on non-standard libraries. I will remove the `requirements.txt` file because of including the test project directly.